cmake/GenerateVersion.cmake

```diff
@@ -33,4 +33,7 @@ endif()
 if(NOT "${NVIM_VERSION_HASH}" STREQUAL "${CURRENT_VERSION_HASH}")
   message(STATUS "Using NVIM_VERSION: ${NVIM_VERSION}")
   file(WRITE "${OUTPUT}" "${NVIM_VERSION_STRING}")
+  if(WIN32)
+    configure_file("${OUTPUT}" "${OUTPUT}" NO_SOURCE_PERMISSIONS @ONLY NEWLINE_STYLE UNIX)
```

Comment:
`NO_SOURCE_PERMISSIONS` is only available from cmake 3.19, so we should probably not have that. I don't think it's needed regardless?

Same with `@ONLY`, I don't think it's needed, although it doesn't hurt either.

Reply:
I removed `NO_SOURCE_PERMISSIONS` (it was supposed to be `USE_SOURCE_PERMISSIONS`; I don't know how I copy-pasted the wrong thing when it's in capital letters).  Keeping `@ONLY` unless you confirm for remove.

Reply:
Done.